Source/WebCore/css/process-css-properties.py

```diff
@@ -743,6 +758,7 @@ def perform_fixups(self, all_properties):
         self.perform_fixups_for_synonyms(all_properties)
```

Comment:
Shouldn't we remove the code for synonyms?

Reply:
yeah, later